components/utils/cachedrequestloader/src/main/java/org/mozilla/cachedrequestloader/CachedRequestLoader.java

```diff
@@ -30,7 +30,7 @@ public CachedRequestLoader(Context context, String subscriptionKey, String subsc
     }
 
     public CachedRequestLoader(Context context, String subscriptionKey, String subscriptionUrl, String userAgent, int socketTag) {
-        this(context, subscriptionKey, subscriptionUrl, userAgent, socketTag, false);
+        this(context, subscriptionKey, subscriptionUrl, userAgent, socketTag, true);
```

Comment:
This probably breaks the test. Please call the false version in the test.

Reply:
Actually, I'm wondering why you're not calling the other constructor. Also, is this intended?